### PR TITLE
166350744-chore(deployment): delopy to heroku

### PR DESCRIPTION
**Pull request by willeswa**
### What does this PR do?
Sets up and deploy the base application to Heroku

### Description of Task to be completed?

  - [x] Create and set up production app on heroku
  - [x] Setup the apps to deploy from the correct branches
  - [x] Provision Heroku Postgres database for both
### How should this be manually tested?
 After the first successful deployment, the prod app should be accessible [here](https://wg-baqi-prod.herokuapp.com/api/v2/)
### Any background context you want to provide?
 No deployment has been done as yet so the apps are not yet live.

### What are the relevant pivotal tracker stories?
#166350744

### CODE REVIEW

**Review by jmusila — CHANGES_REQUESTED** [review]
Good job @willeswa, the app is running on Heroku, but was just concerned, what happened to bootstrap?

**Review by ceexon — APPROVED** [review]

**Comment by kburudi** [comment]
@jonathanmusila if I got you right,
I think you just have to create the app once in Heroku.
Once the app exists it won't be necessary to keep running the create-settings and bootstrap-wger commands any more.

**Review by jmusila — APPROVED** [review]

**Review by wangonya — COMMENTED** [review]
@willeswa what happens when you deploy the branch? If everything is working then the app should be live using the code from your branch.

**Comment by willeswa** [comment]
> Good job @willeswa, the app is running on Heroku, but was just concerned, what happened to bootstrap?

That's a valid question. I actually thought it out and decided to bring back bootstrap. I have managed to have the commands run on Heroku and the app builds the same way it would build locally or like on Travis. But I have to push the settings file because Heroku doesn't seem to save it.

**Review by wekesa931 — COMMENTED** [review]
Does this mean that we will change how we run our app such that we do away with the create-settings command? You can then create a .env sample file so that we all be on the same page.

**Comment by willeswa** [comment]
> @willeswa what happens when you deploy the branch? If everything is working then the app should be live using the code from your branch.

This has been solved in the current push.

File: wger/settings.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        'NAME': os.getenv('NAME'),
+        'USER': os.getenv('USER'),
+        'PASSWORD': os.getenv('PASS'),
+        'HOST': os.getenv('HOST'),
+        'PORT': ''
+    }
+}
+
+# Make this unique, and don't share it with anybody.
+SECRET_KEY = '@d-bo*ihpg_mo#64ya+=d050df!gq5bum59qbf4+yh+cn+h3^c'
```
Comment (caveinn) [inline wger/settings.py]: It is discouraged to have sensitive information such as this hard coded to the system. I would encourage you to change it to an env variable

**Review by caveinn — CHANGES_REQUESTED** [review]

File: wger/settings.py (hunk 1 above)
Comment (willeswa) [reply wger/settings.py]: Resolved

**Comment by caveinn** [comment]
LGTM

**Review by caveinn — APPROVED** [review]

**Pull request merged**